knowledge-base/dropdownbutton-add-separator-between-items.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Adding a Separator Between Items in a DropDownButton for Blazor
+description: Learn how to add a separator between the items in a DropDownButton by utilizing custom styling in a Blazor application.
+type: how-to
+page_title: How to Implement Item Separators in a Telerik Blazor DropDownButton
+slug: dropdownbutton-kb-add-separator-between-items
+tags: dropdownbutton, blazor, item separator, custom styling
+res_type: kb
+ticketid: 1680307
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>DropDownButton for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+I want to visually separate the items in a [DropDownButton](slug:dropdownbutton-overview) to enhance the UI and improve user experience. How to add a separator (divider) between the items?
+
+## Solution
+
+The DropDownButton does not have a concept for a "separator" item (similar to the [ContextMenu](slug:contextmenu-data-binding-overview), for example). However, the DropDownButton uses a declarative approach for its items, so you can add any desired item.
```

Comment:
Start with a positive statement - how to achieve the desired result. If it's important to confirm something negative, you can add that later.

Reply:
```suggestion
The DropDownButton uses a declarative approach for its items, so you can add any desired item. This component does not have a concept for a "separator" item (similar to the [ContextMenu](slug:contextmenu-data-binding-overview), for example) but you can use a custom approach to create a separator.
```